### PR TITLE
chore: test translations

### PR DESCRIPTION
**Pull request by gabrielfs7**
https://oat-sa.atlassian.net/browse/ADF-438

### CODE REVIEW

**Review by chinnu-focaloid — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: model/import/Report/ReportBuilder.php [hunk 1]
```diff
@@ -18,49 +18,198 @@
...
 
-        $report = new Report($type, $outerHeader, $resource);
-        $report->add(Report::createSuccess($innerHeader, $resource));
+    public function buildByException(Throwable $exception): Report
+    {
+        $report = Report::create(Report::TYPE_ERROR, 'CSV import failed');
+        $report->add(
+            Report::create(
+                Report::TYPE_ERROR,
+                'An unexpected error occurred during the CSV import. The system returned the following error: `%s`',
```
Comment (boajer) [inline model/import/Report/ReportBuilder.php:118]: As we have a tool, which is responsible for collecting list of translatable phrases (`tao/scripts/taoTranslate.php`) it should be modified to be aware of such interpolated messages

**Review by boajer — CHANGES_REQUESTED** [review]
Looks good, but IMO require modification
https://github.com/oat-sa/extension-tao-itemqti/pull/1773/files#r649973796

**Review by boajer — APPROVED** [review]
Comments may be addressed in scope of another PR, not blocker

File: model/import/Report/ReportBuilder.php (hunk 1 above)
Comment (gabrielfs7) [reply model/import/Report/ReportBuilder.php:125]: I agree and we created another task for this. Thanks :)

**Comment by AntonNeviarouski** [comment]
QA verified on http://tabular-adf-156-items-import.playground.kitchen.it.taocloud.org:46001/

**Pull request merged**